### DISCUSSION TITLE
A few more keyboard shortcuts?

### DISCUSSION THREAD

**Discussion by git-ansik** [pakerwreah/Calendr#203] [Ideas]
Hey! 
I'm a heavy keyboard user, would love to be able to interact more with only keyboard, few suggestions
- Enter to open selected day in native calendar (as an alternative to double-clicking)
- Cmd+left/right to change month
- Go to today / current month
- Move focus down to events and reminders
- Pin calenadar
- Open Reminders / Open calendar
- Global Shortcut to open the next / current event menu bar item 

Love and hugs

**Comment by git-ansik** [comment]
Very happy with all of these, this is quickly turing out to be the perfect menu bar calendar!

I haven’t figured out GitHub yet so I’ll just post the new translation below (with some slight alternations since last)

/* 
  Localizable.strings
  Calendr

  Created by Gustav (git-ansik) on 22/07/2024.
*/

"quit" = "Avsluta";
"search" = "Sök";
"new_version" = "Ny version tillgänglig på: %@";

"settings.title" = "Inställningar";
"settings.tab.general" = "Allmänt";
"settings.tab.calendars" = "Kalendrar";
"settings.tab.keyboard" = "Genvägar";
"settings.tab.about" = "Om";

"settings.menu_bar" = "Menyrad";
"settings.menu_bar.auto_launch" = "Starta vid inloggning";
"settings.menu_bar.show_icon" = "Visa ikon";
"settings.menu_bar.show_date" = "Visa datum";
"settings.menu_bar.show_icon_date" = "Visa datum i ikon";
"settings.menu_bar.show_background" = ”Visa opak bakgrund";
"settings.menu_bar.date_format_custom" = "Anpassad";
"settings.next_event" = "Nästa händelse";
"settings.menu_bar.show_next_event" = "Visa nästa händelse";
"settings.menu_bar.font_size" = "Teckenstorlek";
"settings.menu_bar.next_event_detect_notch" = "Förkorta om 'notch' finns";

"settings.calendar" = "Kalender";
"settings.calendar.show_week_numbers" = "Visa veckonummer";
"settings.calendar.preserve_selected_date" = "Bevara valt datum när app göms";
"settings.calendar.show_declined_events" = "Visa avböjda händelser";
"settings.calendar.show_declined_events_tooltip" = ”Fungerar endast om det också är aktiverat i den inbyggda kalenderappen.";

"settings.events" = "Händelser";
"settings.events.finished" = "Avslutade";
"settings.events.finished.fade" = "Tona";
"settings.events.finished.hide" = "Dölj";

"settings.transparency" = "Transparens";

"settings.keyboard.local_shortcuts.title" = "App genvägar";
"settings.keyboard.local_shortcuts.prev_date" = "Föregående dag";
"settings.keyboard.local_shortcuts.next_date" = "Nästa dag";
"settings.keyboard.local_shortcuts.prev_week" = "Föregående vecka";
"settings.keyboard.local_shortcuts.next_week" = "Nästa vecka";
"settings.keyboard.local_shortcuts.prev_month" = "Föregående månad";
"settings.keyboard.local_shortcuts.next_month" = "Nästa månad";
"settings.keyboard.local_shortcuts.curr_date" = "Dagens datum";
"settings.keyboard.local_shortcuts.open_date" = "Öppna vald dag";
"settings.keyboard.local_shortcuts.pin_calendar" = "Fäst kalender";

"settings.keyboard.global_shortcuts.title" = "Globala genvägar";
"settings.keyboard.global_shortcuts.show_calendar" = "Visa kalender";

"formatter.date.all_day" = "Heldag";
"formatter.date.today" = "Idag";
"formatter.date.relative.in" = "om %@";
"formatter.date.relative.ago" = "%@ sedan";
"formatter.date.relative.left" = "%@ kvar";

"reminder.options.button" = "Alternativ";
"reminder.options.complete" = "Klar";
"reminder.options.remind" = "Påminn %@";

"event_details.participant.organizer" = "arrangören";
"event_details.participant.me" = "jag";

"event_status.label" = "min status:";

"event_status.accepted" = "Godkänd";
"event_status.maybe" = "Kanske";
"event_status.declined" = "Avböjd";
"event_status.pending" = "Väntande";

"event_action.open" = "Öppna";
"event_action.skip" = "Skippa";
"event_action.accept" = "Tacka ja";
"event_action.maybe" = "Kanske";
"event_action.decline" = "Avböj";

**Reply by pakerwreah** [reply]
Hahaha too late, I've already translated everything 😁
The new version is out as well, with more global shortcuts.
